File: frontend/rentACar/src/app/features/rentals/models/carModels/createdCarDto.ts
import { CarState } from "src/app/core/models/enum/carState";

export interface CreatedCarDto {
  modelId:number;
  coloId:number;
  plate :string;
  kilometer : number;
  modelYear:number;
  rentalBranchId : number;
  CarState :CarState;
  minFindeksCreditRate : number;

}

File: src/rentACar/Application/Features/Cars/Dtos/CreatedCarDto.cs
﻿using Domain.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.Features.Cars.Dtos
{
    public class CreatedCarDto
    {
        public int Id { get; set; }
        public int ColorId { get; set; }
        public int ModelId { get; set; }
        public int RentalBranchId { get; set; }
        public CarState CarState { get; set; }
        public int Kilometer { get; set; }
        public short ModelYear { get; set; }
        public string Plate { get; set; }
        public short MinFindeksCreditRate { get; set; }
    }
}
